rfc/text/advanced/ap_10_rpc_progressive_calls.md

```diff
@@ -120,21 +120,9 @@ set `"progress": false` which is the default.
 
 RPCs can have a multiple registrations (see `Shared Registration` feature) with different `<invocation_policies>`.
 In this case, progressive `CALL` messages can be routed to different *Callees*, which can lead to unexpected results.
```

Comment:
"can be routed" is confusing because it leads one to believe that this is allowed. I suggest this:

> However, allowing progressive `CALL` messages to be routed to different *Callees* would lead to unexpected results.

Reply:
Not late. I added it. Thnx!